cmd/kyverno/main.go

```diff
@@ -122,16 +123,13 @@ func main() {
 
 	cleanUp := make(chan struct{})
 	stopCh := signal.SetupSignalHandler()
+	kubeconfig = ""
```

Comment:
Why are we resetting `kubeconfig` var ?

Reply:
Oh sorry actually I just reverted the changes made in the following [PR code](https://github.com/kyverno/kyverno/pull/3802/files#diff-91bd8f815658713afb6f4ffef18067b5473481b4dd2eee02a6e9c12ec96e384fL75) , I'll just remove it.